### DISCUSSION TITLE
Rootless pass through only some interfaces

### DISCUSSION THREAD

**Discussion by namazso** [containers/podman#15298] [Q&A] [answered]
Is it possible to only pass through only **some** interfaces to a rootless container? I know that --net=host works, but it passes through all interfaces.

**Comment by rhatdan** [comment]
@giuseppe @mheon @Luap99 PTAL
I am not sure if you can do this by setting up a new network namespace and then passing it into a container.  If this was possible, it would be a nice feature.

**Comment by mheon** [comment, marked as answer]
I don't think this is possible without a setuid helper to make us a network namespace that matches the desired configuration, unfortunately.
